src/js/routes/services.js

```diff
@@ -1,37 +1,60 @@
 import Router from 'react-router';
 let Route = Router.Route;
 
+import DeploymentsTab from '../pages/services/DeploymentsTab';
 import ServiceOverlay from '../components/ServiceOverlay';
 import ServicesPage from '../pages/ServicesPage';
+import ServicesTab from '../pages/services/ServicesTab';
 
 let servicesRoutes = {
-  type: Route,
-  name: 'services',
-  path: 'services/?',
-  handler: ServicesPage,
-  children: [
-    {
-      type: Route,
-      name: 'services-detail',
-      path: ':id/?'
-    },
-    {
-      type: Route,
-      name: 'service-ui',
-      path: 'ui/:serviceName/?',
-      handler: ServiceOverlay
-    },
-    {
-      type: Route,
-      name: 'services-panel',
-      path: 'service-detail/:serviceName/?'
-    },
-    {
-      type: Route,
-      name: 'services-task-panel',
-      path: 'task-detail/:taskID/?'
-    }
-  ]
+  services: {
+    type: Route,
+    name: 'services',
+    path: 'services/?',
+    handler: ServicesPage,
+    children: [
+      // Undocumented RR 0.13.5 behaviour: unnamed routes are treated as default
+      {
+        type: Route,
+        handler: ServicesTab
+      },
+      {
+        type: Route,
+        name: 'services-deployments',
+        path: 'deployments/',
+        handler: DeploymentsTab
+      }
+    ]
+  },
+  serviceDetails: {
```

Comment:
Could we do something similar to what we do here to not add another top level route?
https://github.com/dcos/dcos-ui/pull/56/files#diff-cd50a5d0c3afa7ada47e360dc9000693R41